### PR TITLE
sync TrainReader.batch to global train_batch_size

### PR DESCRIPTION
**Pull request by heavengate**
**sync TrainReader.batch to global train_batch_size**
- remove `batch_size` config from `YOLOv3Loss`

### CODE REVIEW

File: ppdet/core/workspace.py [hunk 1]
```diff
@@ -97,6 +97,14 @@ def load_config(file_path):
         del cfg[READER_KEY]
 
     merge_config(cfg)
+
+    # NOTE: training batch size defined only in TrainReader, sychornized
+    #       batch size config to global, models can get batch size config
+    #       from global config when building model.
+    #       batch size in evaluation or inference can also be added here
+    global_config['train_batch_size'] = global_config['TrainReader'][
+        'batch_size']
```
Comment (qingqing01) [inline ppdet/core/workspace.py:106]: If ‘TrainReader’ is not in global_config, will fail

File: configs/yolov3_reader.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ TrainReader:
               [30, 61], [62, 45], [59, 119],
               [116, 90], [156, 198], [373, 326]]
     downsample_ratios: [32, 16, 8]
-  batch_size: 8
+  batch_size: 12
```
Comment (qingqing01) [inline configs/yolov3_reader.yml:49]: 12?

File: ppdet/core/workspace.py (hunk 1 above)
Comment (heavengate) [reply ppdet/core/workspace.py]: Done, thanks!

File: configs/yolov3_reader.yml (hunk 2 above)
Comment (heavengate) [reply configs/yolov3_reader.yml]: Done, thanks!

**Review by still-wait — APPROVED** [review]

**Pull request merged**